src/scenekit.cs

```diff
@@ -594,6 +598,20 @@ interface SCNCamera : SCNAnimatable, SCNTechniqueSupport, NSCopying, NSSecureCod
 		[Export ("screenSpaceAmbientOcclusionNormalThreshold")]
 		nfloat ScreenSpaceAmbientOcclusionNormalThreshold { get; set; }
 
+#endif
+#if !XAMCORE_4_0
+		// Inlined from SCNAnimatable protocol, this allows us to provide an API to be used with ISCNAnimationProtocol
+		// while XAMCORE_4_0 happens and we are able to fix the protocol itself.
+		[Mac (10,13), iOS (11,0), TV (11,0), Watch (4,0)]
+		[Sealed]
+		[Export ("addAnimation:forKey:")]
+		void AddAnimation (ISCNAnimationProtocol scnAnimation, [NullAllowed] string key);
+
+		// Needed to avoid error CS0535: `SceneKit.SCNCamera' does not implement interface member `SceneKit.ISCNAnimatable.AddAnimation(CoreAnimation.CAAnimation, Foundation.NSString)'
+		[NoWatch]
+		[Mac (10,8), iOS (8,0)]
+		[Export ("addAnimation:forKey:")]
+		new void AddAnimation (CAAnimation animation, NSString key);
```

Comment:
hmm... that's quite a bit of duplicated code being generated.

Can't we have a (internal) binding using an `IntPtr` and then manual wrappers for both versions ? I guess not because it's a protocol :( that might be a place where default interface implementation would help.